src/cartservice/Program.cs

```diff
@@ -41,6 +46,9 @@ class ServerOptions
 
             [Option('r', "redis", HelpText = "The ip of redis cache")]
             public string Redis { get; set; }
+
+            [Option("projectId", HelpText = "The ProjectId to which telemetry will flow")]
```

Comment:
Right now no other service requires `projectID` parameter/variable.

They all have OC SD Exporter libraries that are capable of discovering whether it's running on GCE, and querying projectID from the GCE metadata service.